### PR TITLE
Release v2.0.0.1

### PR DESCRIPTION
**Pull request by BenjaminRueth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+
+## Requirements
+
+MATLAB R2018a or later is required. The bindings were tested on R2018b, R2019a and R2019b.
+
+## Restrictions
+
+- An issue causes MATLAB to crash upon SolverInterface initialization if precice was compiled with openmpi. This issue can be resolved by installing openmpi from source using the option `-disable-dlopen`. For reference, see e.g. [here](https://stackoverflow.com/questions/26901663/error-when-running-openmpi-based-library). Alternatively, the user can switch to a different MPI implementation, e.g. MPICH (other implementations were not tested). Note that for [using a different MPI implementation](https://github.com/precice/precice/wiki/Building:-Using-CMake#build-precice-using-non-default-mpi-implementation) one has to specify the alternative implementation while building preCICE.
+- The API function `getMeshHandle` is missing. Also, there are no wrapper classes for MeshHandle etc.
```
Comment (uekerman) [inline README.md]:
```suggestion
```
`MeshHandle` was removed in preCICE with v2.0

File: README.md [hunk 2]
```diff
@@ -1,2 +1,68 @@
...
+- Currently, only one instance of the `SolverInterface` class can exist at the same time in a single MATLAB instance. If the user wishes to couple multiple participants based on MATLAB, he is supposed to start them in different MATLAB instances. If, for some reason, the user needs multiple instances of `SolverInterface`, he should use the OOP variant (Multiple instances of `SolverInterfaceOOP` can exist at the same time).
```
Comment (uekerman) [inline README.md]:
```suggestion
- Currently, only one instance of the `SolverInterface` class can exist at the same time in a single MATLAB instance. If the user wishes to couple multiple participants based on MATLAB, he is supposed to start them in different MATLAB instances. If, for some reason, the user needs multiple instances of `SolverInterface`, he should use the OOP variant (Multiple instances of `SolverInterfaceOOP` can exist at the same time, see further below).
```

File: solverdummy/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# MATLAB solverdummy
+
+This dummy illustrates the use of precice with MATLAB using the MATLAB bindings based on the C data API.
+
+**Please note:** Currently, the MATLAB solverdummy can't be coupled with other solvers unless an exchange directory is specified with its full path in the config file. So, if you wish to couple with a different solver, please make sure to modify the config file accordingly.
```
Comment (uekerman) [inline solverdummy/README.md]:
```suggestion
**Please note:** Currently, the MATLAB solverdummy can't be coupled with other solvers unless an exchange directory is specified with its absolute path in the config file. So, if you wish to couple with a different solver, please make sure to modify the config file accordingly.
```

Comment (uekerman) [reply solverdummy/README.md]: Does `full` mean `absolute`? Why is this the case anyway? Sounds a bit fishy.

File: solverdummy/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# MATLAB solverdummy
```
Comment (uekerman) [inline solverdummy/README.md]: Shouldn't the complete solverdummy folder be in `examples`?

Comment (BenjaminRueth) [reply solverdummy/README.md:1]: For the https://github.com/precice/python-bindings we also provide the solverdummies directly under `/solverdummies`.

I'm fine with leaving everything as it is, but for consistency putting the solverdummies into `examples/solverdummies` would make sense.

Comment (fsimonis) [reply solverdummy/README.md:1]: Putting them into `examples` would be consistent with precice itself. So I think this should be the way to go for all bindings.

**Pull request merged**